library/kong

```diff
@@ -38,62 +56,6 @@ GitFetch: refs/tags/1.5.1
 Directory: centos
 Architectures: amd64
 
-Tags: 1.4.3-alpine, 1.4.3, 1.4
-GitCommit: d2884ee222e72c3edb9abd48d4062991e41ea7bc
-GitFetch: refs/tags/1.4.3
-Directory: alpine
-Architectures: amd64
-
-Tags: 1.4.3-ubuntu, 1.4-ubuntu
-GitCommit: d2884ee222e72c3edb9abd48d4062991e41ea7bc
-GitFetch: refs/tags/1.4.3
-Directory: ubuntu
-Architectures: amd64, arm64v8
-
-Tags: 1.4.3-centos, 1.4-centos
-GitCommit: d2884ee222e72c3edb9abd48d4062991e41ea7bc
-GitFetch: refs/tags/1.4.3
-Directory: centos
-Architectures: amd64
-
-Tags: 1.3.1-alpine, 1.3.1, 1.3
-GitCommit: 48b240c47a4d902ddcc20de408e2c08855a5feca
-GitFetch: refs/tags/1.3.1
-Directory: alpine
-Architectures: amd64
-
-Tags: 1.3.1-ubuntu, 1.3-ubuntu
-GitCommit: 48b240c47a4d902ddcc20de408e2c08855a5feca
-GitFetch: refs/tags/1.3.1
-Directory: ubuntu
-Architectures: amd64, arm64v8
-
-Tags: 1.3.1-centos, 1.3-centos
-GitCommit: 48b240c47a4d902ddcc20de408e2c08855a5feca
-GitFetch: refs/tags/1.3.1
-Directory: centos
-Architectures: amd64
-
-Tags: 1.2.3-alpine, 1.2.3, 1.2
-GitCommit: f5512d2898dad8ad6c95d42b1762ea004713d519
-GitFetch: refs/tags/1.2.3
-Directory: alpine
-
-Tags: 1.2.3-centos, 1.2-centos
-GitCommit: f5512d2898dad8ad6c95d42b1762ea004713d519
-GitFetch: refs/tags/1.2.3
-Directory: centos
-
-Tags: 1.1.3-alpine, 1.1.3, 1.1
-GitCommit: 196331b1e6b4798032af4d6c218a441e2c8db74d
-GitFetch: refs/tags/1.1.3
-Directory: alpine
-
-Tags: 1.1.3-centos, 1.1-centos
-GitCommit: 196331b1e6b4798032af4d6c218a441e2c8db74d
-GitFetch: refs/tags/1.1.3
-Directory: centos
-
 Tags: 1.0.4-alpine, 1.0.4, 1.0
```

Comment:
Let's also remove these 1.0.4 from the list.

Reply:
Should we delay this PR or can this removal be addressed in another PR?